resources/tribe-events-ajax-calendar.js

```diff
@@ -20,12 +20,15 @@
 
 		var $body = $( 'body' ),
 			$nav_link = $( '[class^="tribe-events-nav-"] a' ),
-			base_url = '/',
 			initial_date = tf.get_url_param( 'tribe-bar-date' ),
 			$tribedate = $( '#tribe-bar-date' ),
 			date_mod = false;
 
-		if ( $nav_link.length ) {
+		var base_url = '/';
+
+		if ( 'undefined' !== typeof TribeCalendar.events_base ) {
+			base_url = TribeCalendar.events_base;
+		} else if ( $nav_link.length ) {
```

Comment:
Only use the link failover if we don't have a `TribeCalendar.events_base` for some strange reason.